src/test/java/seedu/address/logic/commands/AddTransactionCommandTest.java

```diff
@@ -88,6 +89,23 @@ public void execute_unfilteredList_success() {
         assertCommandSuccess(addTransactionCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
```

Comment:
could you copy this test to the other commands as well? Then it would be good to merge

Reply:
Added! Please help to review again thanks